includes/size-analysis/prerequisites.mdx

```diff
@@ -0,0 +1,4 @@
+To upload builds you'll need:
+
+1. A Sentry account with access to any [core plan](https://sentry.io/pricing/)
+2. A Sentry auth token – [generate one here](https://sentry.sentry.io/settings/auth-tokens/) and set `SENTRY_AUTH_TOKEN` in your environment so the tools can pick it up automatically
```

Comment:
This is our org, right? Shall we maybe embed a snippet where you can generate the auth token right in the docs, like [here](https://docs.sentry.io/cli/configuration/#to-authenticate-manually)?

<img width="938" height="140" alt="Google Chrome 2025-10-21 09 55 13" src="https://github.com/user-attachments/assets/f9b845ee-f4be-44de-9c47-5e99db07301d" />